internal/pkg/block/block.go

```diff
@@ -33,9 +33,9 @@ type Block struct {
 	// ElectionProof is the vrf proof giving this block's miner authoring rights
 	ElectionProof *crypto.ElectionProof
 
-	// DrandEntries contain the verifiable oracle randomness used to elect
+	// BeaconEntries contain the verifiable oracle randomness used to elect
 	// this block's author leader
-	DrandEntries []*drand.Entry
```

Comment:
I'd prefer the nomenclature less tightly coupled to DRAND throughout. I played with renaming the `drand` package to `beacon` but it was too much noise in this change.

Reply:
Yes this is a good idea. I'll file an issue